incident_viewer_demo/incident/incident.go

```diff
@@ -41,6 +45,9 @@ type DefaultIncident struct {
 	badEndTime       time.Time
 	avgGoodDS        float64
 	avgBadDS         float64
+	//the next field is technically ASN but has to start with low case letter to be private
```

Comment:
Just make the entire variable name lowercase so that it looks better and is consistent with the other names